docs/modules/ROOT/pages/api/erc1155.adoc

```diff
@@ -228,6 +239,28 @@ NOTE: {src5-component-required-note}
 * xref:#ERC1155Component-URI[`++URI(value, id)++`]
 --
 
+[#ERC1155Component-Hooks]
+==== Hooks
+
+Hooks are functions which implementations can extend the functionality of the component source code. Every contract
+using ERC1155Component is expected to provide an implementation of the ERC1155HooksTrait. For basic token contracts, an
+empty implementation with no logic must be provided.
```

Comment:
Should we highlight `ERC1155Component` and/or `ERC1155HooksTrait` or do you think they're good as is?

Reply:
We might add a link to the ERC1155HooksTrait, but since we don't have a doc entry for it, I don't think it is worth it.

I created a page in notion for style guidelines for as to start adding an following the rules in there, and I added rule No. 1 for doc-site being module names shouldn't be `coded` unless the are code units (e. g. use clauses).